gradio/components/image.py

```diff
@@ -8,7 +8,8 @@
 
 import numpy as np
 from gradio_client.documentation import document, set_documentation_group
-from PIL import Image as _Image  # using _ to minimize namespace pollution
+from PIL import Image as _Image
+from PIL import ImageOps  # using _ to minimize namespace pollution
```

Comment:
nit: This comment should be in the previous line.

Reply:
Yea I think black actually did this cause I originally placed both imports in the same line and it complained about length. Will fix!